src/Resources/views/Order/Shop/_downloadButton.html.twig

```diff
@@ -0,0 +1,8 @@
+{% import '@SyliusUi/Macro/buttons.html.twig' as buttons %}
+
+{{ buttons.default(
+    path('sylius_refund_shop_credit_memo_download', {'id': credit_memo.id}),
+    'sylius_refund.ui.download'|trans,
```

Comment:
There is an inconsistency between `shop` and `admin` buttons. Here we have applied `|trans` filter. I wasn't sure which way is correct

Reply:
There is no need for `|trans` filter here, as it is used on message argument in the `default` macro function